Assets/Global/Scripts/Enemies/damagePopUp.cs

```diff
@@ -0,0 +1,44 @@
+using TMPro;
+using UnityEngine;
+
+public class damagePopUp : MonoBehaviour
+{
+    private TextMeshPro textMesh;
+    private float showDamage = 0;
+    public float duration;
+    public float activeDuration;
+    public Color defaultColor;
```

Comment:
can be private / hidden in editor. Setting it in editor doesnt change anything, as the color set as Vertex Color in the text settings override it (might be the expected/preferred behavior anyways).

Reply:
changed activeduration to private

Reply:
think it was specifically, about the defaultColor.
![image](https://github.com/user-attachments/assets/54fde312-d114-45b8-aa10-eaaf5d7b8150)